src/convertToObject.js

```diff
@@ -3,16 +3,37 @@
 /**
  * Implement convertToObject function:
  *
- * Function takes a string with styles (see an example in [stylesString.js](./stylesString.js))
+ * Function takes a string with styles
+ * (see an example in [stylesString.js](./stylesString.js))
  * and returns an object where CSS properties are keys
- * and values are the values of related CSS properties (see an exampl in [test file](./convertToObject.test.js))
+ * and values are the values of related CSS properties
+ * (see an exampl in [test file](./convertToObject.test.js))
  *
  * @param {string} sourceString
  *
  * @return {object}
  */
 function convertToObject(sourceString) {
-  // write your code here
+  const arrayOfProperties = sourceString.split(';');
+  const clearedProperties = arrayOfProperties
+    .map(property => {
+      return property.replace(';', '').replace(/\n/g, '').trim();
+    })
+    .filter(propery => propery);
+  let styleObject = {};
+
+  for (const property of clearedProperties) {
+    const keyAndValue = property.split(':');
+
+    keyAndValue[0] = keyAndValue[0] && keyAndValue[0].trim();
```

Comment:
you can use shorthand
```suggestion
    keyAndValue[0] = keyAndValue[0]?.trim();
```